ledger/metrics/src/main/scala/com/daml/metrics/api/opentelemetry/OpenTelemetryFactory.scala

```diff
@@ -31,12 +31,14 @@ trait OpenTelemetryFactory extends Factory {
       name: MetricName
   )(implicit
       context: MetricsContext = MetricsContext.Empty
-  ): MetricHandle.Timer =
+  ): MetricHandle.Timer = {
+    val nameWithSuffix = name :+ "duration" :+ "ms"
```

Comment:
Nit: In order to comply with the [Prometheus naming rules](https://prometheus.io/docs/practices/naming/#metric-names) (even if this is not `seconds` yet), wouldn't we want to suffix this with `millis` or `milliseconds` instead?

Reply:
we plan on changing to seconds so it's irrelevant for the milliseconds

Reply:
OK, as long as we'll be switching to a standard metric name it's fine.